src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -214,11 +217,26 @@ export class ViewMappingProfile extends Component {
 
     const isMARCRecord = existingRecordType === MARC_TYPES.MARC_BIBLIOGRAPHIC;
 
+    const defaultFieldMappingForMARCColumns = ['action', 'field', 'indicator1', 'indicator2',
+      'subfield', 'subaction', 'data', 'position'];
+    const updatesFieldMappingForMARCColumns = ['field', 'indicator1', 'indicator2', 'subfield'];
```

Comment:
As far as those constants are used only in `renderMARCTableView()` consider moving them into the method.

Reply:
done